From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Buildroot <buildroot@buildroot.org>
Date: Sun, 29 Sep 2026 00:00:00 +0000
Subject: [PATCH] cgpt: include sys/sysmacros.h for major() macro

glibc 2.28+ (Ubuntu 20.04+) requires sys/sysmacros.h for major()/minor()
macros. Include it to fix implicit declaration warnings and link errors.

Signed-off-by: Buildroot <buildroot@buildroot.org>
---
 cgpt/cgpt_wrapper.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cgpt/cgpt_wrapper.c b/cgpt/cgpt_wrapper.c
index 1111111..2222222 100644
--- a/cgpt/cgpt_wrapper.c
+++ b/cgpt/cgpt_wrapper.c
@@ -18,6 +18,7 @@
 #include <string.h>
 #include <sys/stat.h>
 #include <sys/types.h>
+#include <sys/sysmacros.h>
 #include <unistd.h>

 #include "cgpt.h"
--
2.20.1
